control/navigation.c

```diff
@@ -234,6 +234,7 @@ static void navigation_set_speed_command(float rel_pos[], navigation_t* navigati
 	
 	dir_desired_bf[2] = rel_pos[2];
 	
+	norm_rel_dist = max(navigation->cruise_speed,norm_rel_dist);
```

Comment:
norm_rel_distance is a distance and navigation->cruise_speed is a speed...
strange conversion there... isn't it ?